libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/telemetry/CachedNavigationFeedbackEvent.kt

```diff
@@ -0,0 +1,53 @@
+package com.mapbox.navigation.core.internal.telemetry
+
+import com.mapbox.navigation.core.telemetry.events.FeedbackEvent
+
+/**
+ * A cache version of an internal MapboxTelemetry feedback event.
+ * It holds the required data for Feedback UI to display and to be updated with user input.
+ *
+ * @param feedbackId the key of the feedback
+ * @param feedbackType one of [FeedbackEvent.Type]
+ * @param description the user's additional comment about the feedback (optional)
+ * @param screenshot encoded screenshot (optional)
+ * @param feedbackSubType array of [FeedbackEvent.Description] (optional)
+ */
+class CachedNavigationFeedbackEvent internal constructor(
```

Comment:
I believe this can be now a `data class` instead of a `Builder`. BTW if we want to leave it as is, why are we missing the `toString`? cc @LukasPaczos

Reply:
`data class` exposes a public `.copy()` method. If someone relies on that method, their build will break when we add a new constructor parameter, even if the constructor itself is `internal`.

Reply:
The class itself is `internal` so that shouldn't be a problem, but as I mentioned I'm fine with `Builder` too, as long as we add the `toString` 😅

Reply:
Oh I didn't notice you moved it to `internal` package :+1: if it was public, it wouldn't have to have a builder pattern anyway, just a `class CachedNavigationFeedbackEvent internal constructor` with `equals/hash/toString` overrides.

Reply:
Yeah, that's correct.

Reply:
I meant "builder" 😅